pulser-simulation/pulser_simulation/simconfig.py

```diff
@@ -144,6 +144,10 @@ def from_noise_model(cls: Type[T], noise_model: NoiseModel) -> T:
         if "amplitude" in noise_model.noise_types:
             kwargs.setdefault("laser_waist", float("inf"))
         kwargs.pop("with_leakage", None)
+        if "eff_noise_opers" in kwargs:
+            kwargs["eff_noise_opers"] = list(
+                map(qutip.Qobj, kwargs["eff_noise_opers"])
+            )
```

Comment:
Here it doesn't matter if they are dense or CSR?

Reply:
Normally not, I have handled the conversion in the Hamiltonian class. I let the possibility to define dense Qobj in the NoiseModel, so I think there can be dense Qobj in the SimConfig as well :)